exercises.py

```diff
@@ -1,28 +1,53 @@
-"""
-Recursion Problem Set: Coding Exercises
-"""
+"""Recursion Problem Set: Coding Exercises"""
 
 def reverse(s: str) -> str:
-    """
-    Reverse string using recursive approach
-    """
 
+    if len(s) == 0 or len(s) == 1:
+        return s
+    else:
+        return reverse(s[1:]) + s[0]    
+        
 def fib_rec(n: int) -> int:
-    """
-    Recursive fibonacci
-    """
+    """Recursive fibonacci"""
+    if n == 1 or n == 0:
+        return 1
```

Comment:
I think you meant to return `n` here, fib(0) = 0. Be sure to run the tests. Not seeing that you removed the `skip` decorators